testing/e2e/skr-tester/pkg/broker/client.go

```diff
@@ -13,10 +14,9 @@ import (
 )
 
 const (
-	scope                    = "broker:write"
-	KymaServiceID            = "47c9dcbf-ff30-448e-ab36-d3bad66ba281"
-	trialPlanID              = "7d55d31d-35ae-4438-bf13-6ffdfa107d9f"
-	defaultExpirationSeconds = 600
+	scope         = "broker:write"
+	KymaServiceID = "47c9dcbf-ff30-448e-ab36-d3bad66ba281"
+	trialPlanID   = "7d55d31d-35ae-4438-bf13-6ffdfa107d9f"
```

Comment:
Could we refer to constants defined in `plans.go`?

Reply:
I have tried to avoid dependencies on KEB. Currently the application has no dependencies, but I can change it if you see that it's more profitable.